sdk/keyvault/azure-security-keyvault-administration/swagger/autorest.md

```diff
@@ -0,0 +1,52 @@
+# Azure Key Vault Administration for Java
+> see https://aka.ms/autorest
+
+### Setup
+Increase max memory if you're using Autorest older than 3. Set the environment variable `NODE_OPTIONS` to `--max-old-space-size=8192`.
+
+This is the AutoRest configuration file for KeyVaultAccessControlClient.
+---
+## Getting Started 
+To build the SDK for KeyVaultAccessControlClient, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+### Generation
+There is one swagger for KeyVault Administration: rbac. It uses the following tag: `--tag=rbac-preview`.
+
+```ps
+cd <swagger-folder>
+autorest --use=@microsoft.azure/autorest.java@4.0.0 --tag=${package} 
+```
+
+e.g.
+```ps
+cd <swagger-folder>
+autorest --use=@microsoft.azure/autorest.java@4.0.0 --tag=rbac-preview  
+```
+
+## Code generation settings
+``` yaml
+java: true
+output-folder: ../
+namespace: com.azure.security.keyvault.administration
+license-header: MICROSOFT_MIT_SMALL
+models-subpackage: implementation.models
+custom-types-subpackage: models
+generate-client-as-impl: true
+sync-methods: none
+add-context-parameter: true
+context-client-method-parameter: true
+```
+
+### Tag: rbac-preview
+These settings apply only when `--tag=rbac-preview` is specified on the command line.
+
+``` yaml $(tag) == 'rbac-preview'
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/rbac.json
+title: KeyVaultAccessControlClient
```

Comment:
How will you handle other clients?

Reply:
You can add more tags with different configuration properties to `autorest.md`. You can the specify which tag you want to use to generate code. Here's [an example](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/search/azure-search-documents/swagger/readme.md#set-odatametadata-accept-header-in-operations).